### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by shadow-27**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
   <title>Welcome to potionify</title>
   </head>  
 <body>
+  <h1>Welcome to potionify! Enjoy your stay.</h1>
```
Comment (su-codes) [inline index.html]: Could you remove the 'Enjoy your stay' string and make the 'Welcome to potionify ! ' string in red

Comment (shadow-27) [reply index.html]: sure! can do

Comment (su-codes) [reply index.html]: great!

**Review by su-codes — APPROVED** [review]
Well done Shadow !

**Pull request merged**